### DISCUSSION TITLE
v0.10.0

### DISCUSSION THREAD

**Discussion by ManifoldFR** [Simple-Robotics/aligator#252] [Announcements]
## What's Changed
* Change from deprecated `ConstraintSetBase<T>` to `ConstraintSetTpl<T>` by @ManifoldFR in https://github.com/Simple-Robotics/aligator/pull/233
* Add a multibody friction cone cost, rename `FrictionConeResidual` to `CentroidalFrictionConeResidual` by @edantec in https://github.com/Simple-Robotics/aligator/pull/234
* Add gravity compensation residual by @ManifoldFR in https://github.com/Simple-Robotics/aligator/pull/235
* Add pixi support by @jorisv in https://github.com/Simple-Robotics/aligator/pull/240
* Fixes to ProxDDP by @ManifoldFR in https://github.com/Simple-Robotics/aligator/pull/242
* Change linear friction cone to ice cream cone by @edantec in https://github.com/Simple-Robotics/aligator/pull/238
* topic/gar/devel by @ManifoldFR in https://github.com/Simple-Robotics/aligator/pull/236
* Update pixi lockfile by @update-pixi-lockfile in https://github.com/Simple-Robotics/aligator/pull/245
* Fix build instruction by @jorisv in https://github.com/Simple-Robotics/aligator/pull/249
* Fully support building aligator without Pinocchio support (including without proxsuite-nlp Pinocchio support) by @ManifoldFR in https://github.com/Simple-Robotics/aligator/pull/250
* Switch to nonmonotone linesearch by @ManifoldFR in https://github.com/Simple-Robotics/aligator/pull/244

## New Contributors
* @update-pixi-lockfile made their first contribution in https://github.com/Simple-Robotics/aligator/pull/245

**Full Changelog**: https://github.com/Simple-Robotics/aligator/compare/v0.9.0...v0.10.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Simple-Robotics/aligator/releases/tag/v0.10.0'>v0.10.0</a>.</em>

**Comment by Wayne-Z** [comment]
memo: In example/centroidal.py, aligator.FrictionConeResidual() should be synchronised to aligator.CentroidalxFrictionConeResidual()

**Reply by ManifoldFR** [reply]
We actually ended up fixing this in 1996f06589a20, thanks !
